src/hardware/vga_other.cpp

```diff
@@ -906,19 +906,26 @@ void VGA_SetupOther(void) {
 		extern Bit8u int10_font_14[256 * 14];
 		for (i=0;i<256;i++)	memcpy(&vga.draw.font[i*32],&int10_font_14[i*14],14);
 		vga.draw.font_tables[0]=vga.draw.font_tables[1]=vga.draw.font;
-		MAPPER_AddHandler(CycleHercPal,MK_f11,0,"hercpal","Herc Pal");
+		MAPPER_AddHandler(CycleHercPal, SDL_SCANCODE_F11, 0,
+		                  "hercpal", "Herc Pal");
 	}
 	if (machine==MCH_CGA) {
 		IO_RegisterWriteHandler(0x3d8,write_cga,IO_MB);
 		IO_RegisterWriteHandler(0x3d9,write_cga,IO_MB);
-		if(!mono_cga) {
-			MAPPER_AddHandler(IncreaseHue,MK_f11,MMOD2,"inchue","Inc Hue");
-			MAPPER_AddHandler(DecreaseHue,MK_f11,0,"dechue","Dec Hue");
-			MAPPER_AddHandler(CGAModel,MK_f11,MMOD1|MMOD2,"cgamodel","CGA Model");
-			MAPPER_AddHandler(Composite,MK_f12,0,"cgacomp","CGA Comp");
+		if (!mono_cga) {
+			MAPPER_AddHandler(IncreaseHue, SDL_SCANCODE_F11, MMOD2,
+			                  "inchue", "Inc Hue");
+			MAPPER_AddHandler(DecreaseHue, SDL_SCANCODE_F11, 0,
+			                  "dechue", "Dec Hue");
```

Comment:
The increase - decrease feels excessive to step through just a small number of hues. 
What do you think about using one key instead and calling it 'Next Hue' or 'Cycle Hue'?

Reply:
Not a blocker for this PR, but curious what you think!

Reply:
Ah; not sure what to do with these two - they do not cycle through a small number of preset hues - they incrementally shift the hue one way or another…

It's a bit like changing colour for monochrome mode, except it affects CGA composite mode only :/

Reply:
Ahh; then it's perfect, and likely something I haven't used yet!  (I had misunderstood it as cycling through amber, green, white, a Paperwhite.)

Reply:
Minor correction in my last comment.. I though it cycled the CGA palettes (cyan/purple to red/orange). 

I forgot that composite mode's channels can be subtly tweaked to change how they overlap with eachother (with notable effect). I haven't used this in real life either.. but looking at the result, they are very pleasing and impressive for being pre-EGA!